src/LAD4_1/lad4_1.cpp

```diff
@@ -1,5 +1,176 @@
+#include "apogeeDetection.h"
 #include <Arduino.h>
+#include <batteryMonitor.h>
+#include "Barometer.h"
+#include "config.h"
+#include "IMU.h"
+#include "GPS.h"
 
-int main(int argc, char** argv) {
-  return 0;
+#define SERIAL_INPUT 0
+
+#if SERIAL_INPUT
+  #define RFSerial Serial
+#else
+  #define RFSerial Serial6
+#endif
+
+#define GPS_Serial Serial8
+
+uint8_t val_index = 0;
+char command[75];
+
+//Stores how often we should be requesting data from each sensor.
+int sensor_checks[numSensors][2];
+
+sensorInfo *sensor;
+
+long startTime;
+String packet;
+
+void sensorReadFunc(int id);
+
+IMU _imu;
+Barometer bmp;
+GPS gps(GPS_Serial);
+ApogeeDetection detector;
+
+double acc_z;
+double altitude;
```

Comment:
why are these two here. we shouldn't need any of this here. all of that should go inside the apogee detection library.

Reply:
@ssalcedo00